game_patch/rf/particle_emitter.h

```diff
@@ -136,7 +136,7 @@ struct ParticleEmitter
     float time_to_change;
     float current_state_time;
     bool active;
-    int field_144;
+    int active_distance;;
```

Comment:
Double colon. Also is there a way to set this field in editor?

Reply:
Yes.